server/config.py

```diff
@@ -137,7 +137,9 @@ def __init__(self):
         self.GEO_IP_DATABASE_MAX_AGE_DAYS = 22
 
         self.LADDER_1V1_OUTCOME_OVERRIDE = True
-        self.LADDER_ANTI_REPETITION_LIMIT = 2
+        self.LADDER_ANTI_REPETITION_LIMIT = 1
```

Comment:
You said there were issues with this being `2`. Add a unit test which: enforces this being 1 until the underlying issue is fixed. Add a short description to the unit test of what the issue was.